app/views/app_groups/show.html.slim

```diff
@@ -76,5 +76,4 @@
 br
 br
 
-- if @allow_see_apps
```

Comment:
why are we removing this check?

Reply:
the reason is because everyone who has access to see this information page can see apps, is it not?